src/Microsoft.SymbolStore.Client/StoreQueryBuilder.cs

```diff
@@ -17,14 +19,29 @@ internal static class StoreQueryBuilder
 
         public static readonly string SourcePrefix = "sha1";
 
-        public static string GetPortablePdbQueryString(Guid guid, uint stamp, string fileName)
+        public static string GetPortablePdbQueryString(string filename, Guid guid, int age)
         {
-            return fileName + "/" + PortablePdbPrefix + guid.ToString("N") + stamp.ToString("x8") + "/" + fileName;
+            Debug.Assert(filename == Path.GetFileName(filename));  // parts of the path should have already been stripped
+            Debug.Assert(age >= 0);
+
+            string result = $"{filename}/{PortablePdbPrefix + guid.ToString("N") + age.ToString()}/{filename}";
+            return result;
+        }
+
+        public static string GetWindowsPdbQueryString(string filename, Guid guid, int age)
```

Comment:
Its probably more accurate to call this a 'Microsoft', 'Microsoft C++' or 'MSF' pdb.